packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -289,6 +293,13 @@ class SearchAnchor extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.textCapitalization}
   final TextCapitalization? textCapitalization;
 
+  /// Invoked upon user input on the search view.
```

Comment:
The next callback says "called when" so it's worth using the same terms here. That said, I'm not sure "user input on the search view" means. When the user selects a search result? Maybe you mean:
```
/// Called each time the user modifies the search view's text field.
```

This callback and then next one should have "see also" links that point at each other.

Reply:
Updated. Thanks!!